modules/core/build.gradle.kts

```diff
@@ -197,6 +197,10 @@ publishing {
             }
         }
     }
+
+    repositories {
+        mavenLocal()
```

Comment:
This should be removed. Using the local maven repository leads to all kind of issues (see https://github.com/ACINQ/lightning-kmp/blob/master/settings.gradle.kts#L10), devs can manually enable it in `settings.gradle.kts` if needed.

Reply:
I don't think it's needed, using a local maven repo would be needed only when experimenting with SNAPSHOT versions of `secp256k1-kmp` or `bitcoin-kmp`, people who got there will know how to modify their build scripts (and to test for significant changes we publish SNAPSHOT versions of these libraries anyway).